zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -34,24 +36,26 @@ sealed trait Server[-R, +E] { self =>
 object Server {
   private[zhttp] final case class Settings[-R, +E](
     http: HttpApp[R, E] = HttpApp.empty(Status.NOT_FOUND),
-    port: Int = 8080,
     leakDetectionLevel: LeakDetectionLevel = LeakDetectionLevel.SIMPLE,
     maxRequestSize: Int = 4 * 1024, // 4 kilo bytes
     error: Option[Throwable => ZIO[R, Nothing, Unit]] = None,
     sslOption: ServerSSLOptions = null,
+    address: InetSocketAddress = new InetSocketAddress(8080),
   )
 
   private final case class Concat[R, E](self: Server[R, E], other: Server[R, E])      extends Server[R, E]
-  private final case class Port(port: Int)                                            extends UServer
   private final case class LeakDetection(level: LeakDetectionLevel)                   extends UServer
   private final case class MaxRequestSize(size: Int)                                  extends UServer
   private final case class App[R, E](http: HttpApp[R, E])                             extends Server[R, E]
   private final case class Error[R](errorHandler: Throwable => ZIO[R, Nothing, Unit]) extends Server[R, Nothing]
   private final case class Ssl(sslOptions: ServerSSLOptions)                          extends UServer
+  private final case class Address(address: InetSocketAddress)                        extends UServer
 
   def app[R, E](http: HttpApp[R, E]): Server[R, E]                                   = Server.App(http)
   def maxRequestSize(size: Int): UServer                                             = Server.MaxRequestSize(size)
-  def port(int: Int): UServer                                                        = Server.Port(int)
+  def port(port: Int): UServer                                                       = Server.Address(new InetSocketAddress(port))
+  def address(hostname: String, port: Int): UServer                                  = Server.Address(new InetSocketAddress(hostname, port))
+  def address(inetAddress: InetAddress, port: Int): UServer                          = Server.Address(new InetSocketAddress(inetAddress, port))
```

Comment:
```suggestion
  def bind(port: Int): UServer                                                       = Server.Address(new InetSocketAddress(port))
  def bind(hostname: String, port: Int): UServer                                  = Server.Address(new InetSocketAddress(hostname, port))
  def bind(inetAddress: InetAddress, port: Int): UServer                          = Server.Address(new InetSocketAddress(inetAddress, port))
```

Reply:
Changing the function name will be a breaking change. Is it fine?